VERSION "1.0"

NS_ :
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: PCM1 FOO

BO_ 496 ExampleMessage: 8 PCM1
 SG_ Temperature : 7|12@0- (0.01,250) [229.53|270.47] "degK" Vector__XXX, FOO
 SG_ AverageRadius : 1|6@0+ (0.1,0) [0.0|5.0] "m" Vector__XXX
 SG_ Enable : 0|1@0+ (1.0,0) [0.0|0.0] "-" Vector__XXX

CM_ BO_ 496 "Example message used as template in MotoHawk models.";
CM_ SG_ 496 AverageRadius "";








